packages/ripple/src/ripple-directive.ts

```diff
@@ -19,6 +19,7 @@ import MDCRippleFoundation from '@material/ripple/foundation.js';
 import {MDCRippleAdapter} from '@material/ripple/adapter.js';
 import {style} from './mwc-ripple-global-css.js';
 import * as util from '@material/ripple/util.js';
```

Comment:
Looks like only `supportsCssVariables` is imported now. Can you clean that up?

Reply:
done